documentation/writing_tests.md

```diff
@@ -422,6 +424,77 @@ rush add --dev -p @azure/test-utils-recorder
+- The resources created in the tests should be unique. Running the same test in parallel, multiple times, should not break them.
+
+You can read more recomendations through the following link: [Best Practices for writing tests that target live resources](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/51/Testing-Guidelines).
+
+### CI configuration files
+
+To ensure that our tests are executed in the test [Azure DevOps pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) that have been previously configured by our team, some configuration files are necessary:
+
+- A file named `ci.yml` will be automatically generated at the common parent of the clients of a specific service (let's say, at the `keyvault/` level of `keyvault/keyvault-keys`).
+- A `test-resources.json` file, also at the common parent of the clients of a service, which will contain an [ARM template](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview) of the resources needed to run all of the tests of these clients.
+- And a `tests.yml` file located at the folder of each one of the clients, which is in charge of specifying what package to run, what environments to use to run the tests, and how to run the tests.
+
+To explore how to write these files, we recommend going through the guide: [Creating live tests](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/48/Create-a-new-Live-Test-pipeline?anchor=creating-live-tests).
+
+### Activating the pipeline
+
+When adding a new client to our repository, you must report to the _Engineering Systems_ team what client are you adding. They will make sure that the pipelines are running for your project, though you can take more time to finish up your live test setup. After following up with them, once you've merged your tests' code and the configuration files needed, the pipelines will be almost ready to run these test. The next step is to adjust them to send the correct information to your tests.
+
+Go to https://dev.azure.com/azure-sdk/ and look for the `internal` builds that have been configured to target your project, then:
+
+- Click the pipeline you want to test on PR.
+- Press the "Edit" button at the top right corner.
+- Press the three dot menu at the top right corner. A menu will drop down. Press the "Triggers" option from that menu.
+- A page will load with an horizontal menu near the top-center with the following options: "YAML", "Variables", "Triggers" and "History".
+- Click the Variables option of that menu.
+- Click on "Variable Groups".
+- See if in the "Variable Groups" page there's a group called "Release Secrets for GitHub". If so, you're done.
+- If that group is not visible in that page, add it: A bit below the groups, you'll see a button "🔗 Link variable group". Click it, then use the "🔍 search" input and type "release secrets", and then link the group called "Release Secrets for GitHub".
+
```

Comment:
@sadasant May be we can add in another point about which  Variable Groups to connect to for live tests pipelines - and they would be "Secrets for Resource Provisioner" if they use the resource provisioning through the arm templates

Reply:
Although some also use "Test Secrets for JS Live Tests". They will need to ensure that the secrets they need are present in the variable group they are configuring

Reply:
I think I have addressed this at this point! I will reach out to you to day to make sure I've done what you meant.